assets/js/utilities/notificationUtilities.js

```diff
@@ -4,9 +4,9 @@ define('notificationUtilities', [], function() {
 		return {
 			giveNotice: function(title, message, affirmative = true) {
 				WDN.initializePlugin('notice');
-				const body = angular.element(document.querySelector('body' ));
+				const body = angular.element(document.querySelector('body'));
 				body.append(`
-					<div class="wdn_notice ${affirmative ? 'affirm' : 'negate'}" data-overlay="maincontent" data-duration="10">
```

Comment:
@CollinsSpencer I removed the `data-duration` since it was suggested not to use by the WDN page. Also, I added fixed positioning so that one wouldn't need to jump the page up to the top so a user could see the notification. Thoughts on if this was a good idea?

Reply:
I think this is fine.